spec/helpers/application_helper_spec.rb

```diff
@@ -41,28 +41,28 @@
     end
   end
 
-  describe "#remove_breadcrumbs" do
+  describe "#show_breadcrumbs?" do
```

Comment:
I think perhaps these tests need to be amended to when the content item is a homepage, landing_page or is nil rather than referring to "is_landing_page?"

Reply:
Done 👍